geopyspark/geotrellis/constants.py

```diff
@@ -93,237 +42,207 @@
 """
 ROWMAJOR = 'rowmajor'
 
+"""The NoData value for ints in GeoTrellis."""
+NODATAINT = -2147483648
 
-"""A time unit used with ZORDER."""
-MILLISECONDS = 'millis'
-
-"""A time unit used with ZORDER."""
-SECONDS = 'seconds'
-
-"""A time unit used with ZORDER."""
-MINUTES = 'minutes'
-
-"""A time unit used with ZORDER."""
-HOURS = 'hours'
-
-"""A time unit used with ZORDER."""
-DAYS = 'days'
 
-"""A time unit used with ZORDER."""
-MONTHS = 'months'
+class ResampleMethods(object):
+    """Resampling Methods."""
 
-"""A time unit used with ZORDER."""
-YEARS = 'years'
+    NEARESTNEIGHBOR = 'NearestNeighbor'
+    BILINEAR = 'Bilinear'
+    CUBICCONVOLUTION = 'CubicConvolution'
+    CUBICSPLINE = 'CubicSpline'
+    LANCZOS = 'Lanczos'
+    AVERAGE = 'Average'
+    MODE = 'Mode'
+    MEDIAN = 'Median'
+    MAX = 'Max'
+    MIN = 'Min'
 
 
-"""Neighborhood type."""
-ANNULUS = 'annulus'
+RESAMPLE_METHODS = [
```

Comment:
Can these be moved into the class somehow so they can be references with `if x in ResampleMethods: ...`  ?